dev.Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM node:18-alpine3.17
+FROM node:18
 
 WORKDIR /app
+COPY . .
 
-# Start the application
-CMD ["npm", "run", "start:dev:docker"]
+RUN npm install
+EXPOSE 3000
+ENV START_PROCESS="api"
+
+CMD ["sh", "-c", "if [ \"$START_PROCESS\" = \"api\" ]; then npm run start:api; else npm run start:worker; fi"]
```

Comment:
Simple docker file that can read which process to start based on env variable, that way we dont need two docker files for api and worker